src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
 }
 ```
 
+## Arterial Spin Labeling perfusion data
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    perf/
```

Comment:
What does perf stand for? Perfusion? Is asl the only modality using this? Maybe add a very brief line below describing and pointing where all these data folder labels are summarized

Reply:
Good point, @patsycle

Reply:
I think these folders are the currently used datatype folders: https://github.com/bids-standard/bids-specification/tree/master/src/schema/datatypes

see definition of "datatype" here: https://bids-specification.readthedocs.io/en/latest/02-common-principles.html#definitions

Reply:
@tsalo I saw perf is not yet in these files. I assume we (BEP005) should add some information there as well?

Reply:
Thanks @sappelhoff @tsalo these links are super useful :)

Reply:
Sorry for the delay- yes, updating the schema to incorporate ASL will have to happen before the PR is merged, but we are holding off on that part until the actual content and structure of the BEP have been approved. When the time comes, I can help.

Reply:
Do we need to help with the definition of "perf"? I would not restrict this to ASL, but for now this would only include ASL.